apps/antalmanac/src/lib/course-helpers.ts

```diff
@@ -35,6 +35,26 @@ export function getCourseInfo(SOCObject: WebsocAPIResponse) {
     return courseInfo;
 }
 
+// An arbitrary URL used to check if Peterportal and/or Websoc are live
+const PING_PETERPORTAL_WEBSOC_URL =
+    'https://api-next.peterportal.org/v1/rest/websoc?year=2023&quarter=Spring&sectionCodes=34271';
+
+export async function isWebsocWorking() {
```

Comment:
I don't love this way of checking whether PP/WebSOC is working because we're essentially making a random extra request for them to have to handle before we make the bulk of our requests. It's not a ping. It's extra load for PP (although I don't know if it's non-negligible), and it slows down our schedule load times, especially after #713.

If it really simplifies the process, then it's justified, but I'm not sure if it does.